floor_generator/lib/value_object/database.dart

```diff
@@ -26,6 +29,7 @@ class Database {
           classElement == other.classElement &&
           name == other.name &&
           entities == other.entities &&
+          views == other.views &&
           daoGetters == other.daoGetters &&
```

Comment:
💡We should start using `ListEquality` for comparing lists in the database class as well.

```suggestion
          const ListEquality<Entity>().equals(entities, other.entities) &&
          const ListEquality<View>().equals(views, other.views) &&
          const ListEquality<DaoGetter>()
              .equals(daoGetters, other.daoGetters) &&
```